app/tests/test_authenticate.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import User
+
+from .base import BaseTestCase
+
+
+class AuthenticationTestCase(BaseTestCase):
```

Comment:
[Test class naming convention](https://docs.python.org/3/library/unittest.html#basic-example). Please refer to the other preexisting test files for guidance too.